website/blog/2023-02-21.ferretdb-compare-arrays.md

```diff
@@ -0,0 +1,530 @@
+---
+title: "How MongoDB compares arrays in filtering"
+slug: ferretdb-compare-arrays
+authors: [chi]
+description: In this article, we explore in detail how MongoDB BSON arrays are compared
+in filter operators.
```

Comment:
```suggestion
description: "In this article, we explore in detail how MongoDB BSON arrays are compared in filter operators."
```

I had to make a few changes to make `task blog-dev` possible to run, maybe docusaurus has changed something in the recent versions? Please check that you can run it locally.

The other changes I made were around `keywords`.